internal/manager/run.go

```diff
@@ -242,14 +269,11 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 func setupKonnectNodeAgentWithMgr(
 	c *Config,
 	mgr manager.Manager,
+	konnectAPIClient *konnect.NodeAPIClient,
```

Comment:
How about renaming it to `konnectNodeClient` if we use separate clients for konnect node APIs and konnect license APIs?